invenio_users_resources/services/users/config.py

```diff
@@ -194,10 +195,12 @@ class UsersServiceConfig(RecordServiceConfig, ConfiguratorMixin):
         "avatar": Link("{+api}/users/{id}/avatar.svg"),
         "records_html": Link("{+ui}/search/records?q=user:{id}"),
         "admin_records_html": Link(
-            "{+ui}/administration/records?q=user:{id}&f=allversions", when=can_manage
+            "{+ui}/administration/records?q=parent.access.owned_by.user:{id}&f=allversions",
```

Comment:
looks correct, I think it was copied from above, could you verify if records_html is working correctly or it needs the same fix?

Reply:
Both draft and records need this fix; only `admin_moderation_html` was working correctly.